packages/ember-template-compiler/lib/plugins/transform-attrs-into-args.js

```diff
@@ -27,6 +27,20 @@ export default function TransformAttrsToProps() {
   this.syntax = null;
 }
 
+function isAttrs(node) {
+  if (node.parts[0] === 'attrs') {
+    return true;
+  }
+
+  let [_this, attrs] = node.parts;
```

Comment:
we may have already worked around this generally but just in-case please double check as this destructuring is not what I suspect you think it is:

https://babeljs.io/repl/#?babili=false&evaluate=true&lineWrap=false&presets=es2015%2Creact%2Cstage-2&code=let%20%5Bz%5D%20%3D%20node.parts